tests/array_interoperability_test.py

```diff
@@ -274,6 +274,91 @@ def testJaxToCuPy(self, shape, dtype):
                      z.__cuda_array_interface__["data"][0])
     self.assertAllClose(x, cupy.asnumpy(z))
 
+  @unittest.skipIf(xla_extension_version < 237, "Requires newer jaxlib")
+  @jtu.sample_product(
+    shape=all_shapes,
+    dtype=cuda_array_interface_dtypes,
```

Comment:
You can use `dtype=jtu.dtypes.supported(cuda_array_interface_dtypes)` and then remove the x64 `skipTest` block below.

Reply:
Thanks for the hint! Done.